src/openforms/forms/validators.py

```diff
@@ -44,3 +46,23 @@ def validate_not_deleted(form):
             _("Form is deleted."),
             code="invalid",
         )
+
+
+def validate_form_definition_is_reusable(instance: FormDefinition, **kwargs) -> None:
+    """
+    - Ensure that ``is_reusable`` can only be switched from true to false
+      if the form definition is used in less than 2 formsteps.
+    """
+    if not instance:
+        return
+
+    is_reusable = kwargs.get("is_reusable", instance.is_reusable)
+
+    if is_reusable is False and instance.used_in.count() > 1:
+        raise ValidationError(
+            {
+                "is_reusable": _(
+                    "Cannot change is_reusable from true to false if the form definition is used in more than one formstep."
+                ),
```

Comment:
```suggestion
                "is_reusable": _(
                    "This form definition cannot be marked as 'not-reusable' as it is used in multiple existing forms."
                ),
```
better user feedback.